src/routes/console/project-[project]/auth/user-[user]/updatePrefs.svelte

```diff
@@ -13,6 +13,8 @@
         if (JSON.stringify(prefs) !== JSON.stringify(Object.entries($user.prefs))) {
             if (!!prefs[prefs.length - 1][0] && !!prefs[prefs.length - 1][1]) {
                 arePrefsDisabled = false;
+            } else if (prefs[prefs.length - 1][0] === '' || prefs[prefs.length - 1][1] === '') {
+                arePrefsDisabled = false;
```

Comment:
this allows me enter empty keys, not sure this is what should be allowed 🤔 

![Bildschirm­foto 2023-03-29 um 15 10 12](https://user-images.githubusercontent.com/1759475/228546893-283e08bd-b5c8-43fd-ad91-5edf6adcdf92.png)

Reply:
Whoops. Didn't acciunt for this. Will update soon.